pkg/reconcilers/deploymentconfig.go

```diff
@@ -43,6 +44,12 @@ func GenericDeploymentConfigMutator(existingObj, desiredObj common.KubernetesObj
 	tmpUpdate = DeploymentConfigContainerResourcesReconciler(desired, existing)
 	update = update || tmpUpdate
 
+	tmpUpdate = DeploymentConfigAffinityReconciler(desired, existing)
```

Comment:
Why not Install Only?

Reply:
It could be install only too I guess.
The parameters are inherently configurable so I decided to implement it configurable.